bundle/manifests/nginx-ingress-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,303 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "k8s.nginx.org/v1alpha1",
+          "kind": "NginxIngressController",
+          "metadata": {
+            "name": "nginxingresscontroller-sample"
+          },
+          "spec": {
+            "foo": "bar"
+          }
+        }
+      ]
```

Comment:
not sure why this is not populated

Reply:
I thought it would be populated from the base https://github.com/nginxinc/nginx-ingress-operator/blob/feat/update-to-sdk-1.7.2/config/manifests/bases/nginx-ingress-operator.clusterserviceversion.yaml#L5 🤔

Reply:
it's populated from samples